### ISSUE TITLE
Get the parsed intent from DeeplinkDelegate

### ISSUE THREAD

**Issue by droidekas** [airbnb/DeepLinkDispatch#217]
Hi,

Is there a way to get the parsed intent from DeeplinkDelegate? I am interested in getting the intent for a deeplink and then performing different functions. 
The current deeplink delegate does not seem to be supporting that. I presume this is by design  to keep the API simple.  But is this a valid usecase?

**Comment by droidekas** [comment]
Yes. I currently have an implementation which implements the functionality of a `DeeplinkDelegate` that the library generates to cater to the requirements I have. If the library owners feel it appropriate I'd love to formalize the API a bit more and contribute for the same.

**Comment by okasurya** [comment]
I think this is a great idea. Also, this could be an alternative for android navigation, especially when you have multiple modules on your project. I have created a simple implementation for this, you can check it here: https://github.com/okasurya/DeepLinkDispatch/commit/f95d0d3c3667510b0ea57c2b97ad83a8e653700e

**Comment by Gnzlt** [comment]
This would be really useful

**Comment by ivorsliide** [comment]
Hello guys I been having this issue for the past year hopping it will get into the code, how is this going? how can I help/contribute to make this change happens?
